tests/integration/suite/test_app.py

```diff
@@ -263,6 +265,269 @@ def test_app_custom_values_file(admin_pc, admin_mc):
     wait_for_app_to_be_deleted(client, app)
 
 
+@pytest.mark.nonparallel
```

Comment:
Why are these tests non-parallel?

Reply:
They all change the rancher version so the clobber each other.